docker/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN \
   apt-get update; \
   apt-get install --no-install-recommends -y libxtst6 xvfb xauth x11-apps imagemagick zip; \
   `# libraries needed by JCEF` \
-  apt-get install --no-install-recommends -y libatk1.0-0 libatk-bridge2.0-0 libgbm1 libglib2.0 libxcomposite1 libxkbcommon0 libxrandr2; \
+  apt-get install --no-install-recommends -y libatk1.0-0 libatk-bridge2.0-0 libgbm1 libglib2.0 libxcomposite1 libxkbcommon0 libxrandr2 sudo; \
```

Comment:
Nit: I'd move it to line 36, current placing suggests that it's needed specifically for JCEF (line 37)

Reply:
Or even better, to line 50 where all such "outlier" packages are located